### DISCUSSION TITLE
Glossary section displays non-glossary term during translation.

### DISCUSSION THREAD

**Discussion by Ere-Eye** [WeblateOrg/weblate#11486] [Q&A] [answered]
In the translation interface, the glossary shows a non-glossary term in _italics_ with no corresponding target language in between valid glossary items. I don't understand why it's there.
![image](https://github.com/WeblateOrg/weblate/assets/14802003/433d269a-3196-4369-a913-17486a95ea84)
That term in the middle is not in the source string either.

**Comment by nijel** [comment]
These are glossary term explanations.

**Reply by Ere-Eye** [reply]
I'm not sure if I've made myself clear. I'm trying to understand why the second item is in there. The "战斗" term is not in the glossary nor in the source string being translated.

**Reply by nijel** [reply, marked as answer]
Is the explanation of the glossary term, you will see it if you click on the pencil icon.

**Reply by Ere-Eye** [reply]
Thank you, I knew I must be missing something and this is not a bug! I added developer comments where available when uploading translations via CSV format. That's where it's from.

**Comment by Geeyun-JY3** [comment]
It also seems necessary that the UI label text should explicitly tell users that the text in _italics_ in the Glossary sidebar is the explanation. Could be part of #9082.

**Reply by nijel** [reply]
I've just pushed some minor improvements in cf8e73a6f7ecc6a765b3e9d7619f6f7e1b618b3a.
